onnxruntime/core/providers/openvino/backends/basic_backend.cc

```diff
@@ -142,6 +139,10 @@ BasicBackend::BasicBackend(const ONNX_NAMESPACE::ModelProto& model_proto,
           config["VPU_HW_INJECT_STAGES"] = CONFIG_VALUE(NO);
           config["VPU_COPY_OPTIMIZATION"] = CONFIG_VALUE(NO);
         }
+    #endif
+    #if defined(OPENVINO_2021_4)
```

Comment:
can you change this to an else case
so that you don't need to make another code change when future versions of OpenVINO get released?
currently you have this strictly checking for 2021.4 when we really want to say any versions greater than 2021.3

Reply:
If i put else I will have to repeat the configs in earlier 2021_3 case. Also currently 2020_4 is also activated which is the else case. Will have a relook at this logic when 2020_4 is deprecated.